docs/LwM2M_Client/Nordic/nRF7002DK.md

```diff
@@ -101,7 +105,7 @@ To connect the board:
     ![Anjay set configuration](images/anjay_config_set.PNG "Anjay set configuration"){:style="float: left;margin-right: 1177px;margin-top: 7px; margin-bottom: 17px;"}
 
 
-0. The **nRF7002 DK** connects to the Internet via a Wi-Fi network. Remember to set the credentials of your network using `anjay config set wifi_ssid <value>` and `anjay config set wifi_password <value>`.
+0. The **{{ nRF7002DK_board_name }}** connects to the Internet via a Wi-Fi network. Remember to set the credentials of your network using `anjay config set wifi_ssid <value>` and `anjay config set wifi_password <value>`.
```

Comment:
What about parts that won't exist in other board's docs? Will these be handled using a script too?

Reply:
I think they should. `yaml` do not support nested varibales too....  `{{ nRF7002DK_board_name }}` need to be inserted into newly created variable. As I said in another comment, there is a lot job to do in another task